schunk_sdh/msg/TemperatureArray.msg

```diff
@@ -0,0 +1,2 @@
+Header header
+Temperature[] temperature_list
```

Comment:
I don't have a preference, as we are not using the hardware atm...
I was just throwing in an idea...I leave it to you to decide...maybe @wxmerkt has an opinion here, too?